### ISSUE TITLE
Known good version that works with existing tooling?

### ISSUE THREAD

**Issue by JamesHutchison** [tronprotocol/docker-tron-quickstart#23]
Currently, it looks like the Dockerfile is checking out the "shasta-dev" branch. It seems like maybe the docker images should be tagged in a way you can choose the underlying network/protocol version? For example, I'm trying to deploy a simple hello world contract and I get `CONTRACT_VALIDATE_ERROR`. Maybe trying against an older version of java-tron would work? This would also help, in that if someone pushes a bug to the dev branch anyone trying this out for the first time isn't going to be extremely frustrated by things not working (like my experience has been so far)

**Comment by JamesHutchison** [comment]
Yeah, when I change `git checkout shasta-dev` to `git checkout Odyssey-v3.1.3` I no longer get that error.

**Comment by JamesHutchison** [comment]
Might make sense to tie a version to what's most recently tested and supported by Tron Studio, since the quick start guide uses that.

**Comment by JamesHutchison** [comment]
I'm saying that the tags in docker should be pegged to a version of java-tron. This would help with consistency and reduce unexpected errors when trying to do something as simple as a tutorial. This is part of the quick start process and thus should _always_ work with the documentation provided for quick start. In its current state it does not and the reason why it doesn't is tribal knowledge you have to stumble upon since I'm sure the reason will always be changing.

Another way of putting it, is that I should be able check out a tag from docker and get the exact same image every time. That is not the case right now.

**Comment by sullof** [comment]
Tron Quickstart is not just a _dockerization_ of java-tron. There are many versions that just added features. Pegging its version to java-tron version would be very limiting.
Also, when a new version of java-tron is released and adopted, having a previous version of java-tron does not make much sense. This means that you always want to use last version of TronBox, Tron Quickstart, etc. 
Another thing that you are right about is that if you rebuild the image, you will obtain a different version. But this is because I have published this repo so that people can modify it, not to do their own build expecting to have the same result. Sometimes, we change something in the branch `event_parser` or `shasta-dev` and I build a new version of this image. The goal is to provide a tool that is as good as possible for testing, not to have consistency among versions.
But I understand your point. What about a version command that returns a json with all the versions of the packages that have been used for the image?

**Comment by sullof** [comment]
@JamesHutchison https://github.com/tronprotocol/docker-tron-quickstart/pull/24
